src/org/labkey/test/tests/SampleTypeLinkToStudyTest.java

```diff
@@ -446,6 +448,46 @@ public void testFolderImportExport()
         checker().verifyEquals("No data should have been imported", 0, table.getDataRowCount());
     }
 
+    /*
+        Test coverage for : Issue 42937: Assay results grid loading performance can degrade with a large number of "copied to study" columns
+     */
+    @Test
+    public void testLinkedColumnDisabledCase()
+    {
+        log("Creating 2 more studies");
+        _containerHelper.createProject("Study 1", "Study");
+        _studyHelper.startCreateStudy().createStudy();
+
+        _containerHelper.createProject("Study 2", "Study");
```

Comment:
Does the behavior change for subfolder ? I am curious about the distinct name, any particular reason?

Reply:
Nope, the behavior won't change for projects vs subfolders. I was just suggesting subfolders as a way to put the study folders under the current test project so that it is one less project to need to cleanup on test completion.

Reply:
Changed the name by prefixing the project name.